### PR TITLE
Supplier payment

### PR DESCRIPTION
**Pull request by make-it-git**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #

Notify the following people: @darrynten

### CODE REVIEW

File: src/Models/BankAccount.php [hunk 1]
```diff
@@ -0,0 +1,148 @@
+<?php
```
Comment (darrynten) [inline src/Models/BankAccount.php:1]: Don't forget to tick this one off in the README too :)

**Review by darrynten — COMMENTED** [review]
Is this the PR with models in it that you need help with tests for?

File: src/Models/BankAccount.php (hunk 1 above)
Comment (make-it-git) [reply src/Models/BankAccount.php:1]: These models are incomplete (only fields are described, any additional methods not created) so I think we can tick them once tests are done.

Comment (darrynten) [reply src/Models/BankAccount.php:1]: OK great, I think we should mark them in the Readme with something like `(partial)` next to the name. What do you think?

**Review by darrynten — CHANGES_REQUESTED** [review]
@make-it-git you'll be pleased to know that @igorsergiichuk will be making tests for the rest of the models in here and opening a PR into this one for these tests:

* BankAccount
* BankFeedAccountGroup
* BankFeedAccount
* BankAccountCategory

He's really quick so this will be soon 🚀

@fergusdixon check this too :D

File: src/Models/BankAccount.php (hunk 1 above)
Comment (make-it-git) [reply src/Models/BankAccount.php:1]: @darrynten marked

**Review by darrynten — APPROVED** [review]

**Pull request merged**